Lib/typing.py

```diff
@@ -751,7 +766,9 @@ def open_helper(file: str, mode: MODE) -> str:
     parameters = _flatten_literal_params(parameters)
 
     try:
-        parameters = tuple(p for p, _ in _deduplicate(list(_value_and_type_iter(parameters))))
+        parameters = tuple(
+            p for p, _ in _deduplicate(list(_value_and_type_iter(parameters)))
+        )
```

Comment:
There's no change being made to this line now except the formatting; let's keep it as it is to preserve `git blame`:

```suggestion
        parameters = tuple(p for p, _ in _deduplicate(list(_value_and_type_iter(parameters))))
```